zeppelin-distribution/src/bin_license/LICENSE

```diff
@@ -188,6 +188,7 @@ The following components are provided under the CDDL License.
 
     (CDDL 1.0) javax.activation (javax.activation:activation:jar:1.1.1 - http://java.sun.com/javase/technologies/desktop/javabeans/jaf/index.jsp)
     (CDDL 1.1) Jersey (com.sun.jersey:jersey:jar:1.9 - https://jersey.java.net/)
+    (CDDL 1.1) (GPL2) jersey-core (org.glassfish.jersey.core:jersey-core:2.22.2 - https://jersey.java.net/)
```

Comment:
I'll make that change later and push the amended version back up.

WRT the transitive dependencies:  I'm not very familiar with licensing and had a question:  If there's a main package reference (e.g. org.glassfish.jersey in our case) that is provided under a particular license (e.g. CDDL 1.1) it's not true that the particular license covers all the dependencies of that package?

I'll run through the list individually and add any items that are needed.